### PR TITLE
Use the latest version of sccache if no version is specified

### LINKED ISSUE #25: when no version specificed, download the latest release
Probably should be done here:
https://github.com/Mozilla-Actions/sccache-action/blob/main/src/setup.ts#L23-L33

### PR DESCRIPTION
**Pull request by miles170**
Closes #25 

I'm not using the `@actions/github` package since it requires a token.
But the side effect is that we might encounter an API rate limit like the following:
```
HttpClientError: API rate limit exceeded for 207.254.3.33. (But here's the good news: Authenticated requests get a higher rate limit. Check out the documentation for more details.)
```

### CODE REVIEW

**Comment by sylvestre** [comment]
i landed a change. could you please rebase your patch to fix the conflict on dist/setup/index.js ? thanks

File: src/setup.ts [hunk 1]
```diff
@@ -19,12 +19,19 @@ import {
...
 async function setup() {
-  // TODO:  we can support install latest version by default if version
-  // is not input.
-  const version = core.getInput('version');
-  core.info(`try to setup sccache version: ${version}'`);
+  let version = core.getInput('version');
+  if (version.length === 0) {
+    // If no version is specified, the latest version is used by default.
+    const http = new HttpClient('sccache-action');
+    const json = await http.getJson(
```
Comment (Xuanwo) [inline src/setup.ts:29]: Thanks for the PR! Do you think it's a good idea to visit ` https://github.com/mozilla/sccache/releases/latest` to get the latest tag?

Comment (miles170) [reply src/setup.ts:29]: Is it your concern that the official API may undergo changes over time that could result in it not working properly?

The reason I am using that API is that `@actions/github` requires a token to be provided when using it.
See https://github.com/actions/toolkit/blob/457303960f03375db6f033e214b9f90d79c3fe5c/packages/github/src/internal/utils.ts#L5

Or should we add the `github_token` parameter to the action?

Comment (Xuanwo) [reply src/setup.ts:33]: > Or should we add the `github_token` parameter to the action?

This seems a common practice to accept a `token` input: https://github.com/actions/setup-go/blob/8dbf352f069be09d9a0b567cc1a9d16a5663fc3a/action.yml#L12C3-L14

I think it's good to follow it :100:

**Pull request merged**